cc/mcts_node.cc

```diff
@@ -304,17 +305,37 @@ void MctsNode::IncorporateResults(absl::Span<const float> move_probabilities,
         legal_moves[i] ? policy_scalar * move_probabilities[i] : 0;
 
     edges[i].original_P = edges[i].P = move_prob;
-    // Initialize child Q as current node's value, to prevent dynamics where
-    // if B is winning, then B will only ever explore 1 move, because the Q
-    // estimation will be so much larger than the 0 of the other moves.
+
+    // NOTE: Minigo uses value {-1, 1} from black's perspective
+    //       Leela uses value {0, 1} from current player's perspective (?)
+    //       AlphaGo uses ???
+    //       ELF uses ???
+    //
+    // The initial value of a child's Q is not perfectly understood.
     //
-    // Conversely, if W is winning, then B will explore all 362 moves before
-    // continuing to explore the most favorable move. This is a waste of
-    // search.
+    // There are a couple of general ideas:
+    //   * Init to Parent: Init a new child to it's parent value
+    //      We think of this as saying "The game is probably the same after *any* move
+    //   * Init to Draw (AKA init to zero): Init a new child to 0 AKA "position looks even":
+    //      We tested this in v11, Because this is how we interpretted the original AGZ paper.
+    //      This doesn't make a lot of sense: The losing player tends to explore every move before
+    //      reading a seond one twice.  The winning player tends to read only the top policy move
+    //      because it has much higher value than any other move
+    //   * Init to Parent minus constant AKA FPU (Leela's approach):
+    //      This outperforms init to parent in eval matches. Leela has used for a long time.
+    //   * Init to Loss:
```

Comment:
this is missing a prose explanation, e.g. "This is like saying 'any unexplored subtree is as bad as a loss"

Reply:
done in other PR.